hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -4,7 +4,20 @@
 
 This tutorial explains how to boot your Raspberry Pi from a USB mass storage device such as a flash drive or a USB hard disk. Note that this feature does not work with all USB mass storage devices.
 
-## Program USB host boot mode (Raspberry Pi 3A+ and 3B only)
+See the [documentation](https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/) for the boot sequence and alternative boot modes (Network, USB device, GPIO or SD boot).
+
+Note that "USB Mass Storage Boot" is different from "USB Device Boot Mode", the device boot mode allows a Raspberry Pi connected to a computer to boot using files on that computer.
+
+## Raspberry Pi 1, 2 (V1.1), Compute Module, Zero
+
+**These models are not supported for USB Mass Storage Boot.**
+The boot code for USB is stored in the BCM2837 device only, so the Pi 1 A/B, Pi 2 B (v1.1), and Pi Zero will all require SD cards as they are based on the BCM2835 and BCM2836. This boot code is stored in ROM (except Pi 4B) which by definition cannot be changed.
+
+An alternative is to use the 'special bootcode.bin-only boot mode' as described [here](https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/). This still requires/boots from an SD-card but allows to run on an USB Device.
```

Comment:
I think "run on an USB Device" is a bit vague? The benefit of this 'special bootcode.bin-only boot mode' is that `bootcode.bin` is the _only_ file read from the SD-card.

Reply:
Again, probably not needed, since the main boot modes page includes information about the bootcode.bin only method of booting.

Reply:
I've changed it to: "This still requires/boots from an SD-card, but bootcode.bin is the only file read from the SD-Card."

However; this means that the SD-Card is not read from anymore after the boot process as well?

Reply:
Yes, but the bootcode.bin file has to be the only file on the SD card for this to work. If any of the other boot files are present, then bootcode.bin will read those, and will try to complete the boot process from the SD card.